<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibratedAntForm</class>
 <widget class="QWidget" name="calibratedAntForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot_calibratedAnt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>661</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot_map">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>661</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_zoomCalibratedAnt">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_zoomMap">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>660</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
